### DISCUSSION TITLE
Status 409 Conflict statusType 4 - Creazione/modifica documento in corso

### DISCUSSION THREAD

**Discussion by monossido** [fattureincloud/api#271] [Q&A] [answered]
Hi, when I'm trying to create new pro forma, a new document without a number, I get this error:

```{"error":{"message":"Creazione\\/modifica di un documento gi\\u00e0 in corso. Riprova."}}```

It seems that fattureincloud has some document in "pending", but I don't know how is that possible.
It's not a matter of multiple requests at the same time because I cannot see any new documents at all.

We are using fatture in cloud API v2 since a few months and apparently this is the first time with this error.

Thanks.

**Comment by valmoz** [comment]
Buongiorno @monossido 
le chiedo gentilmente di contattare il supporto clienti sulla pagina principale di Fatture in Cloud, perché ci servono informazioni aggiuntive per risolvere il suo problema.

Alleghi pure il link a questa discussione per velocizzare il tutto.

Qui le modalità di contatto: [Support](https://developers.fattureincloud.it/docs/support)

Grazie
Mauro

**Comment by monossido** [comment, marked as answer]
Grazie

**Comment by monossido** [comment]
Update: no answer from support
problem has resolved by itself (or after edit some invoice...)
